test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlRetryServiceTests.cs

```diff
@@ -618,9 +618,9 @@ public SqlConnectionBuilderWithConnectionInitializationFailure(ISqlConnectionBui
                 _allRetriesFail = allRetriesFail;
             }
 
-            public async Task<SqlConnection> GetSqlConnectionAsync(string initialCatalog = null, CancellationToken cancellationToken = default)
+            public async Task<SqlConnection> GetSqlConnectionAsync(string initialCatalog = null, int? maxPoolSize = null, CancellationToken cancellationToken = default)
```

Comment:
Where are we using the `maxPoolSize `variable?

Reply:
the interface is updated in healthcare-shared-component so this update is needed